whctools/app_settings.py

```diff
@@ -9,3 +9,7 @@
 MEDIUM_REJECT = getattr(settings, "WHCTOOLS_MEDIUM_REJECT", 30)
 
 LARGE_REJECT = getattr(settings, "WHCTOOLS_LARGE_REJECT", 365)
+
+IVY_LEAGUE_ALLIANCE = 937872513
+
+IVY_LEAGUE_ALT_ALLIANCE = 99010193
```

Comment:
It will be good to move these into attributes like the ones above in the file.
While it's not important for us as our alliance number won't change, if anyone wants to use our app, they could override it locally.py
Not urgent just a thought